build/rpmbuild.h

```diff
@@ -103,11 +108,14 @@ rpmds rpmSpecDS(rpmSpec spec, rpmTagVal tag);
 
 /** \ingroup rpmbuild
  * Spec build stages state machine driver.
+ * @param ts		rpm transaction set
  * @param spec		spec file control structure
  * @param buildArgs	build arguments
- * @return		RPMRC_OK on success
+ * @return		0 on success, 1 on build error,
+ *			RPMRC_MISSINGBUILDREQUIRES on missing build
```

Comment:
The return values are bit of a mishmash, some numeral and some RPMRC_ value that's not really an RPMRC afterall. Would be more consistent to return RPMRC_* constants for everything (but keep return type as int because of RPMRC_MISSINGBUILDREQUIRES). That, or just talk about numbers.